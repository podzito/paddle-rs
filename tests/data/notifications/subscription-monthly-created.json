{
  "data": {
    "id": "ntf_01j6h8vbfx1wryhg65nxh43b4m",
    "type": "subscription.created",
    "status": "not_attempted",
    "payload": {
      "data": {
        "id": "sub_01j6h8vajcgzcapjyqqs6n65t0",
        "items": [
          {
            "price": {
              "id": "pri_01j52zfss0hcfxamnma1tf42gt",
              "name": "PodZito Pro",
              "type": "standard",
              "status": "active",
              "quantity": { "maximum": 1, "minimum": 1 },
              "tax_mode": "account_setting",
              "created_at": "2024-08-12T09:42:40.160401Z",
              "product_id": "pro_01j52z87e5sdt3p60mmtcavpgt",
              "unit_price": { "amount": "999", "currency_code": "USD" },
              "updated_at": "2024-08-29T11:35:06.922573Z",
              "custom_data": null,
              "description": "PodZito Pro - Monthly",
              "import_meta": null,
              "trial_period": { "interval": "day", "frequency": 7 },
              "billing_cycle": { "interval": "month", "frequency": 1 },
              "unit_price_overrides": []
            },
            "status": "trialing",
            "product": {
              "id": "pro_01j52z87e5sdt3p60mmtcavpgt",
              "name": "PodZito",
              "type": "standard",
              "status": "active",
              "image_url": null,
              "created_at": "2024-08-12T09:38:32.005Z",
              "updated_at": "2024-08-12T09:38:32.005Z",
              "custom_data": null,
              "description": "Podzito",
              "import_meta": null,
              "tax_category": "standard"
            },
            "quantity": 1,
            "recurring": true,
            "created_at": "2024-08-30T09:11:18.86Z",
            "updated_at": "2024-08-30T09:11:18.86Z",
            "trial_dates": { "ends_at": "2024-09-06T09:11:18.855Z", "starts_at": "2024-08-30T09:11:18.855Z" },
            "next_billed_at": "2024-09-06T09:11:18.855Z",
            "previously_billed_at": null
          }
        ],
        "status": "trialing",
        "discount": null,
        "paused_at": null,
        "address_id": "add_01j6h8tmg9n54yeb1rx5z6qxx5",
        "created_at": "2024-08-30T09:11:18.86Z",
        "started_at": "2024-08-30T09:11:18.855Z",
        "updated_at": "2024-08-30T09:11:18.86Z",
        "business_id": null,
        "canceled_at": null,
        "custom_data": { "customer_id": "2lN8BwhpfGzI5pP7domN3P0DcMU" },
        "customer_id": "ctm_01j535fawggst1gm3qsyanc0mb",
        "import_meta": null,
        "billing_cycle": { "interval": "month", "frequency": 1 },
        "currency_code": "USD",
        "next_billed_at": "2024-09-06T09:11:18.855Z",
        "transaction_id": "txn_01j6h8tfbmt61670vyq9farbyd",
        "billing_details": null,
        "collection_mode": "automatic",
        "first_billed_at": null,
        "scheduled_change": null,
        "current_billing_period": { "ends_at": "2024-09-06T09:11:18.855Z", "starts_at": "2024-08-30T09:11:18.855Z" }
      },
      "event_id": "evt_01j6h8vb276se6a1hyys6t9fdn",
      "event_type": "subscription.created",
      "occurred_at": "2024-08-30T09:11:19.367924Z",
      "notification_id": "ntf_01j6h8vbfx1wryhg65nxh43b4m"
    },
    "occurred_at": "2024-08-30T09:11:19.367924Z",
    "delivered_at": null,
    "replayed_at": null,
    "origin": "event",
    "last_attempt_at": null,
    "retry_at": null,
    "times_attempted": 0,
    "notification_setting_id": "ntfset_01j5akfpdszj2r2prnsjbf0ytt"
  },
  "meta": { "request_id": "fc1170f4-e5bd-4331-9d79-45f60ce4de27" }
}
