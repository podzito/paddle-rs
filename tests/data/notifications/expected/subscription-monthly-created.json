{
  "event_id": "evt_01j6h8vb276se6a1hyys6t9fdn",
  "notification_id": "ntf_01j6h8vbfx1wryhg65nxh43b4m",
  "event": {
    "SubscriptionCreated": {
      "id": "sub_01j6h8vajcgzcapjyqqs6n65t0",
      "customer_id": "ctm_01j535fawggst1gm3qsyanc0mb",
      "billing_cycle": { "frequency": 1, "interval": "month" },
      "next_billed_at": "2024-09-06T09:11:18.855Z",
      "paused_at": null,
      "scheduled_change": null,
      "started_at": "2024-08-30T09:11:18.855Z",
      "custom_data": { "customer_id": "2lN8BwhpfGzI5pP7domN3P0DcMU" },
      "items": [
        {
          "price": {
            "billing_cycle": { "frequency": 1, "interval": "month" },
            "created_at": "2024-08-12T09:42:40.160401Z",
            "custom_data": null,
            "description": "PodZito Pro - Monthly",
            "id": "pri_01j52zfss0hcfxamnma1tf42gt",
            "import_meta": null,
            "name": "PodZito Pro",
            "product_id": "pro_01j52z87e5sdt3p60mmtcavpgt",
            "quantity": { "maximum": 1, "minimum": 1 },
            "status": "active",
            "tax_mode": "account_setting",
            "trial_period": { "frequency": 7, "interval": "day" },
            "type": "standard",
            "unit_price": { "amount": "999", "currency_code": "USD" },
            "updated_at": "2024-08-29T11:35:06.922573Z"
          },
          "product": {
            "created_at": "2024-08-12T09:38:32.005Z",
            "custom_data": null,
            "description": "Podzito",
            "id": "pro_01j52z87e5sdt3p60mmtcavpgt",
            "image_url": null,
            "name": "PodZito",
            "status": "active",
            "tax_category": "standard",
            "type": "standard",
            "updated_at": "2024-08-12T09:38:32.005Z"
          },
          "created_at": "2024-08-30T09:11:18.86Z",
          "next_billed_at": "2024-09-06T09:11:18.855Z",
          "previously_billed_at": null,
          "price_id": null,
          "quantity": 1,
          "recurring": true,
          "status": "trialing",
          "trial_dates": { "starts_at": "2024-08-30T09:11:18.855Z", "ends_at": "2024-09-06T09:11:18.855Z" },
          "updated_at": "2024-08-30T09:11:18.86Z"
        }
      ],
      "status": "trialing",
      "transaction_id": "txn_01j6h8tfbmt61670vyq9farbyd",
      "updated_at": "2024-08-30T09:11:18.86Z"
    }
  },
  "occurred_at": "2024-08-30T09:11:19.367924Z"
}

